src/bundle/Resources/config/value_object_visitors.yml

```diff
@@ -199,6 +200,13 @@ services:
         tags:
             - { name: ezpublish_rest.output.value_object_visitor, type: Exception }
 
+    ezpublish_rest.output.value_object_visitor.symfony.HttpException:
+        parent: ezpublish_rest.output.value_object_visitor.base
+        class: "%ezpublish_rest.output.value_object_visitor.HttpException.class%"
```

Comment:
I think we can drop this and just use class name as service name. I know it breaks pattern in that repo/file but we did it multiple times before.

Reply:
I don't mind.

I was also wondering if the visitor for Exceptions should also/instead be tagged with Throwable.

Reply:
Fixed in 60404e6c.

Reply:
Further fixed in cab14d9.